crates/transpiler/src/passes/discretize_rotations.rs

```diff
@@ -0,0 +1,369 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use pyo3::prelude::*;
+use pyo3::wrap_pyfunction;
+use std::f64::consts::PI;
+
+use qiskit_circuit::dag_circuit::DAGCircuit;
+use qiskit_circuit::operations::{OperationRef, Param, StandardGate};
+use rustworkx_core::petgraph::stable_graph::NodeIndex;
+
+const ROTATION_GATE_NAMES: &[&str; 3] = &["rx", "ry", "rz"];
```

Comment:
As per Matt's comment on some of the other PRs, it might be slightly beneficial to change "const" to "static" here.

```suggestion
static ROTATION_GATE_NAMES: [&str; 3] = ["rx", "ry", "rz"];
```

Reply:
done in [066ded3](https://github.com/Qiskit/qiskit/pull/15321/commits/066ded3f9e884d07037336a87d969f43cb54c8eb)